Gemfile

```diff
@@ -68,8 +72,10 @@ group :development, :test do
   gem 'letter_opener'
   gem 'rspec_junit_formatter'
 
-  gem 'jshint'
+  # not compatible with react_on_rails
+  # gem 'jshint'
```

Comment:
We could just remove this entirely then and look for another hinting/linting alternative

Reply:
I've managed to make this work on the rebased branch, testing it more tonight or tomorrow.